app/waf/test/acceptance/service.controller.acceptance.ts

```diff
@@ -39,7 +39,7 @@ describe('ServiceController', () => {
     const service = await givenServiceData(wafapp, uuid(), {id: uuid()});
 
     let response = await client.get(prefix + '/services').expect(200);
-    expect(response.body.services).to.containDeep([toJSON(service)]);
+    expect(toJSON(service)).to.containDeep(response.body.services[0]);
```

Comment:
Why it doesn't report the error when we run the test locally?

Reply:
I did not rebase the branch, after 'build response' pr is merged just now. Since github does not require travis to run pr build again before merging, it allow us to merge pr with an outdated travis build result.